Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG PIP_VERSION=22.0.2
+ARG PIP_VERSION=24.0
```

Comment:
IIRC, pip changed the HTTP cache location between these two versions, so you'll need to update the `--mount` options for the `RUN` commands that involve pip.

Reply:
That's right, it turns out the path had [moved](https://pip.pypa.io/en/stable/topics/caching/) to http-v2, changed the path, thanks!